Arduino due warning: "pgm_read_byte" redefined

In file included from src/main.cpp:2:0:
lib/aWOT/src/aWOT.h:68:0: warning: "pgm_read_byte" redefined
 #define pgm_read_byte(ptr) (unsigned char)(*ptr)
 
In file included from /home/am/.platformio/packages/framework-arduino-sam/cores/arduino/Arduino.h:31:0,
                 from src/main.cpp:1:
/home/am/.platformio/packages/framework-arduino-sam/cores/arduino/avr/pgmspace.h:102:0: note: this is the location of the previous definition
 #define pgm_read_byte(addr) (*(const unsigned char *)(addr))

--- a/library.properties
+++ b/library.properties
@@ -1,5 +1,5 @@
 name=aWOT
-version=3.2.3
+version=3.2.4
 author=Lasse Lukkari <lasse.lukkari@gmail.com>
 maintainer=Lasse Lukkari <lasse.lukkari@gmail.com>
 sentence=Arduino web server library.

--- a/src/aWOT.cpp
+++ b/src/aWOT.cpp
@@ -891,7 +891,7 @@ int Request::read(uint8_t* buf, size_t size) {
     ret++;
   }
 
-  int read = m_stream->read(buf, (size < m_left ? size : m_left));
+  int read = m_stream->read(buf, (size < (unsigned)m_left ? size : m_left));
   if (read == -1) {
     if (ret > 0) {
       return ret;

--- a/src/aWOT.h
+++ b/src/aWOT.h
@@ -64,10 +64,6 @@
 #define SERVER_MAX_HEADERS 10
 #endif
 
-#ifdef _VARIANT_ARDUINO_DUE_X_
-#define pgm_read_byte(ptr) (unsigned char)(*ptr)
-#endif
-
 #define P(name) static const unsigned char name[] PROGMEM
 
 class StreamClient : public Client {
@@ -76,8 +72,8 @@ class StreamClient : public Client {
 
  public:
   StreamClient(Stream* stream) : s(stream){};
-  int connect(IPAddress ip, uint16_t port){return 1;};
-  int connect(const char* host, uint16_t port){return 1;};
+  int connect(IPAddress, uint16_t){return 1;};
+  int connect(const char*, uint16_t){return 1;};
   size_t write(uint8_t byte){return s->write(byte);};
   size_t write(const uint8_t* buffer, size_t length){return s->write(buffer, length);};
   int available(){return s->available();};

